kodilla-basic-tests/src/main/java/com/kodilla/bank/homework/CashMachine.java

```diff
@@ -0,0 +1,76 @@
+package com.kodilla.bank.homework;
+
+public class CashMachine {
```

Comment:
Nie trzeba dwoch tablic trzymac - liczby ujemne to withdrawals, liczby dodatnie to deposits